app/javascript/packs/new_casa_contact.js

```diff
@@ -0,0 +1,15 @@
+import $ from 'jquery'
+$('document').ready(() => {
+  // inside the app/views/casa_contact/_form, we are trying to only show a field if 'other' is chosen.
+  $("#contact_type").change((e) => {
+    if (e.target.value === 'other') {
+      $(".other-contact-type").removeAttr('hidden')
+      $(".other-contact-type input").attr('required', 'required')
+    } else {
+      $(".other-contact-type").attr('hidden', true)
+      $(".other-contact-type input").removeAttr('required')
```

Comment:
It's great that you watched out for making the field required! 👍 It makes me wonder if we should also handle this in the model validations. At the moment, we have no model validations, as far as I know. What do you think?